tests/end-to-end/external-solution-integration/pkg/testsuite/deploy_lambda.go

```diff
@@ -38,6 +38,7 @@ function sendReq(url, resolve, reject) {
 
 module.exports = { main: function (event, context) {
 	console.log("Received event: ", event);
+    console.log(process.env.GATEWAY_URL.length)
```

Comment:
Do we need this?

Reply:
Yes, this is the easiest way to error out if env variables are not injected as a part of ServiceBinding. Or we could error out if HTTP call in the function fails with non 2xx. @sjanota Can we suggest how can we fix the HTTP call?

Reply:
Actually I added it just for organoleptic verification. The test was never meant to rely on that.

Reply:
I would handle it in the `sendReq` function. If the status is not what is expected reject the promise.

Reply:
As discussed offline, throwing a custom error now.